src/app/static/src/app/components/strategisePage.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div class="strategise container mt-5">
+    <h1 class="h3 text-center">
+      Strategize across regions
+    </h1>
+    <div class="help mt-2 text-center">
+      <a href="#" @click="showDocumentation = !showDocumentation">
+        <info-icon></info-icon>
+        How to use this page
+        <component style="vertical-align: top" :is="documentationChevronComponent"></component>
+      </a>
+      <b-collapse :visible="showDocumentation">
+        <div class="my-1" v-html="'<p>This tool can investigate how different interventions could be ' +
+        'distributed cross wider regions to minimise the overall number of malaria cases whilst achieving local ' +
+        'goals.</p>' +
+        '<p>In some circumstances the next best option might be substantially lower cost. To investigate this the ' +
+        'tool outputs not only the best option but shows which combination of interventions give slightly reduced ' +
+        'budgets. The tool presents 5 potential strategies, allowing the user to determine and explore potential ' +
+        'strategies for vector control across regions.</p>' +
+        '<p>Please see the user manual for more information.</p>'"></div>
+      </b-collapse>
+    </div>
+    <div class="form mt-5">
+      <b-form inline class="justify-content-center" @submit.prevent="submit">
+        <label class="mr-sm-2" for="inline-form-input-budget">Total available budget</label>
+        <span class="icon-small mr-sm-3"
+              v-tooltip="'The total available budget to cover the next 3-years is required to assess the most feasible intervention options across regions'">
+          <help-circle-icon></help-circle-icon>
+        </span>
+        <b-input-group prepend="$USD" class="mb-2 mr-sm-3 mb-sm-0">
+          <b-form-input id="inline-form-input-budget" v-model="currentProject.budget" type="number" min="0"
```

Comment:
The `v-model` here immediately propagates changes in global budget to the project, and therefore to each region via `RootMutation.SetCurrentRegionInterventionSettings`. The alternative is to only do this when the form is submitted - I'm not sure which is preferable.

Reply:
> I think this does make most sense in terms of when it updates - other settings take effect immediately. I'm slightly surprised that works though, since you're not updating the region's setting by setting that value in the project - is it because the form always emits values when it loads, and gets those updated from the current project budget at that point?

Exactly

> We've generally steered clear of directly modifying state values through `v-model`, would usually make use of a mutation instead. That mutation could potentially do the updating of the region's settings, though apparently it's not needed!

Mutation is a good idea - not least because it would make it easier to subsequently change the behaviour by keeping the same mutation but trigger it on form submission itself. Will add.

Reply:
Done in 38def7e